### PR TITLE
Jibri status overhaul

### PR DESCRIPTION
**Pull request by bbaldino**

### CODE REVIEW

**Comment by bbaldino** [comment]
depends on https://github.com/jitsi/jitsi/pull/539, will need a pom update

File: src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt [hunk 1]
```diff
@@ -149,11 +156,7 @@ class XmppApi(
      * Function to update outgoing [presence] stanza with current jibri status.
      */
     private fun updatePresenceStanza(presence: Presence) {
-        val jibriStatus = JibriStatusPacketExt()
-        jibriStatus.status =
-                if (jibriManager.busy()) JibriStatusPacketExt.Status.BUSY
-                else JibriStatusPacketExt.Status.IDLE
-        presence.addExtension(jibriStatus)
+        presence.addExtension(jibriStatusManager.overallStatus.toJibriStatusExt())
```
Comment (bgrozev) [inline src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt]: do you want to use addExtension or setExtension here?

Comment (bbaldino) [reply src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt]: thanks, will use the new `setChildExtension`

Comment (bbaldino) [reply src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt]: oh, actually this is a `Presence` which doesn't have a set

**Pull request merged**